src/main/java/heronarts/lx/parameter/LXListenableNormalizedParameter.java

```diff
@@ -57,4 +58,30 @@ public boolean isMappable() {
     return this.mappable;
   }
 
+  public boolean isWrap() {
+    return this.allowWrap;
+  }
+
+  public LXListenableNormalizedParameter setWrap(boolean allowWrap) {
```

Comment:
I think this stuff should live in the `LXNormalizedParameter` interface. For naming, let's have it mirror `setMappable()` and `isMappable()`, so

So in LXNormalizedParameter:
```
public default boolean isWrappable() {
  return false;
}
```

Then you can leave the setter here in LXListenableNormalizedParameter as I think that will cover all cases where we care about being able to set it to anything else.

Reply:
Adjusted.  See if it looks right after I push changes.